README.md

```diff
@@ -45,6 +45,13 @@ Install development dependencies:
 pip install -e '.[dev]'
 ```
 
+Do codegen (necessary for type checking and tests):
```

Comment:
Is `codegen` common knowledge for Python projects? Either way, I'd consider adding a link here.